docs/_redirects

```diff
@@ -50,3 +50,63 @@
+/data-builder-dummy-data                 /data-builder/dummy-data/
+/data-builder-dummy-data/                /data-builder/dummy-data/
+/ehrql-combining-series-and-frames       /data-builder/ehrql/combining-series-and-frames/
+/ehrql-combining-series-and-frames/      /data-builder/ehrql/combining-series-and-frames/
+/ehrql-examples                          /data-builder/ehrql/examples/
+/ehrql-examples/                         /data-builder/ehrql/examples/
+/ehrql-intro                             /data-builder/ehrql/intro/
+/ehrql-intro/                            /data-builder/ehrql/intro/
+/ehrql-reference                         /data-builder/ehrql/reference/
+/ehrql-reference/                        /data-builder/ehrql/reference/
+/ehrql-set-population-explainer          /data-builder/ehrql/set-population-explainer/
+/ehrql-set-population-explainer/         /data-builder/ehrql/set-population-explainer/
+/ehrql-new-tutorial-1a                   /data-builder/ehrql/tutorial/1a/
+/ehrql-new-tutorial-1a/                  /data-builder/ehrql/tutorial/1a/
+/ehrql-new-tutorial-1b                   /data-builder/ehrql/tutorial/1b/
+/ehrql-new-tutorial-1b/                  /data-builder/ehrql/tutorial/1b/
+/ehrql-new-tutorial-2a                   /data-builder/ehrql/tutorial/2a/
+/ehrql-new-tutorial-2a/                  /data-builder/ehrql/tutorial/2a/
+/ehrql-new-tutorial-3a                   /data-builder/ehrql/tutorial/3a/
+/ehrql-new-tutorial-3a/                  /data-builder/ehrql/tutorial/3a/
+/ehrql-new-tutorial-4a                   /data-builder/ehrql/tutorial/4a/
+/ehrql-new-tutorial-4a/                  /data-builder/ehrql/tutorial/4a/
+/ehrql-new-tutorial-5a                   /data-builder/ehrql/tutorial/5a/
+/ehrql-new-tutorial-5a/                  /data-builder/ehrql/tutorial/5a/
+/ehrql-new-tutorial-6a                   /data-builder/ehrql/tutorial/6a/
+/ehrql-new-tutorial-6a/                  /data-builder/ehrql/tutorial/6a/
+/ehrql-new-tutorial-7a                   /data-builder/ehrql/tutorial/7a/
+/ehrql-new-tutorial-7a/                  /data-builder/ehrql/tutorial/7a/
+/ehrql-new-tutorial-conclusion           /data-builder/ehrql/tutorial/conclusion/
+/ehrql-new-tutorial-conclusion/          /data-builder/ehrql/tutorial/conclusion/
+/ehrql-new-tutorial-intro                /data-builder/ehrql/tutorial/intro/
+/ehrql-new-tutorial-intro/               /data-builder/ehrql/tutorial/intro/
+/ehrql-new-tutorial-opensafely-project   /data-builder/ehrql/tutorial/opensafely-project/
+/ehrql-new-tutorial-opensafely-project/  /data-builder/ehrql/tutorial/opensafely-project/
+/ehrql-new-tutorial-python               /data-builder/ehrql/tutorial/python/
+/ehrql-new-tutorial-python/              /data-builder/ehrql/tutorial/python/
+/data-builder-intro                      /data-builder/intro/
+/data-builder-intro/                     /data-builder/intro/
+/query-engine-explanation                /data-builder/query-engine-explanation/
```

Comment:
Strange: this redirect and the ones below it don't appear to work correctly. They do if you move them to the top of the file.

We're nowhere near the [limits](https://developers.cloudflare.com/pages/platform/redirects) of number of redirects, or the character limit per redirect. They also show up in Cloudflare's list of redirects for the deployment.

![cloudflare-redirects](https://user-images.githubusercontent.com/3818079/207689173-21f43f0b-7036-4eab-b332-6eaed22a4dee.png)

Reply:
Just checked and this is still a problem, so :envelope: to support it is.

It's all the URLs below the `query-engine-explanation` line.

Reply:
I've sent Cloudflare an email.

Reply:
If there's no quick fix, I'd be for merging this sooner rather than later anyway to avoid any further dealing with merge conflicts.

Although I would like to know what the issue with the redirects actually _is_.

The affected URLs aren't in widespread public use.

Reply:
It seems like it's the order that's the issue, so if you put all the 302 redirects ahead of 301, then it works :man_shrugging: